src/py42/util.py

```diff
@@ -72,3 +72,30 @@ def format_dict(dict_, label=None):
     if label:
         return u"{} {}".format(label, indented_dict)
     return indented_dict
+
+
+def bool_to_str(value):
+    if isinstance(value, bool):
+        return "true" if value else "false"
+    else:
+        raise KeyError("Value must be True or False")
+
+
+def str_to_bool(value):
+    if value == "true":
```

Comment:
These should call `lowercase` first

Reply:
These bool converters are for the funky-ness of the `OrgSettings` API. The `value` field of the t_settings packets that are submitted for changes (e.g. `"key": "org-securityTools-enable", "value": "false", "locked": false}`). So rather than make the end-user of py42 set the attribute as the strings `true` or `false` we're auto-converting them to booleans when returning the attribute and auto-converting them back to strings when setting them.

I suppose since the method is so specific to these packet values, I could make them methods on the class so they won't be used elsewhere and we can just assume that the inputs are always going to be what we expect.

Reply:
It doesn't hurt! It just always seems like the right thing to do. I figured it wasn't necessary in this case.